### ISSUE TITLE
Add Silent Mode Feature During Friday Salah Time

### ISSUE THREAD

**Issue by burhanaksendir** [NedaaDevs/nedaa#67]
It would be highly beneficial to have a feature that automatically sets the phone to silent mode during the Friday salah time on Fridays. This feature would ensure that users can attend the prayer without disturbances from their phones.

Thank you for considering this feature request.

**Comment by F2had** [comment]
> It would be highly beneficial to have a feature that automatically sets the phone to silent mode during the Friday salah time on Fridays. This feature would ensure that users can attend the prayer without disturbances from their phones.
> 
> 
> 
> Thank you for considering this feature request.

Yeah this is actually nice, I thought about it before on Android it should be easy to implement however, on ios I am not sure if it is possible. If you know a way then please do share. Also, you are more than welcome to work on any issue or translate the app in your language.

**Comment by burhanaksendir** [comment]
> > It would be highly beneficial to have a feature that automatically sets the phone to silent mode during the Friday salah time on Fridays. This feature would ensure that users can attend the prayer without disturbances from their phones.
> > Thank you for considering this feature request.
> 
> Yeah this is actually nice, I thought about it before on Android it should be easy to implement however, on ios I am not sure if it is possible. If you know a way then please do share. Also, you are more than welcome to work on any issue or translate the app in your language.

Thanks.  I wanted to let you know that I've started working on implementing this feature, and if successful, I'll submit it as a Pull Request. It seems feasible to achieve this on Android using system timing and notification controls.

For iOS, due to platform restrictions, we may need to explore different approaches, such as scheduled Do Not Disturb settings or notifications.

**Comment by F2had** [comment]
> > > It would be highly beneficial to have a feature that automatically sets the phone to silent mode during the Friday salah time on Fridays. This feature would ensure that users can attend the prayer without disturbances from their phones.
> > > Thank you for considering this feature request.
> > 
> > 
> > Yeah this is actually nice, I thought about it before on Android it should be easy to implement however, on ios I am not sure if it is possible. If you know a way then please do share. Also, you are more than welcome to work on any issue or translate the app in your language.
> 
> Thanks. I wanted to let you know that I've started working on implementing this feature, and if successful, I'll submit it as a Pull Request. It seems feasible to achieve this on Android using system timing and notification controls.
> 
> For iOS, due to platform restrictions, we may need to explore different approaches, such as scheduled Do Not Disturb settings or notifications.

Sounds great, thanks for the effort. my idea was not limited to Friday only but actually for all prayers. I am not sure but we can have two options one for Friday and another per prayer (example: Fajr only). 
One more thing is how long it will be silent for?
This also might be an option to let the user decide.
